src/integrations/blocks/breadcrumbs-block.php

```diff
@@ -86,11 +86,14 @@ public function present( $attributes ) {
 		$presenter = new Breadcrumbs_Presenter();
 		// $this->context_memoizer->for_current_page only works on the frontend. To render the right breadcrumb in the
 		// editor, we need the repository.
-		if ( defined( 'REST_REQUEST' ) && REST_REQUEST === true ) {
-			$indexable = $this->indexable_repository->find_by_id_and_type( filter_input( INPUT_GET, 'post_id' ), 'post' );
-			$context   = $this->context_memoizer->get( $indexable, 'Post_Type' );
+		if ( ( defined( 'REST_REQUEST' ) && REST_REQUEST === true ) || \is_admin() ) {
```

Comment:
I'd prefer seperate functions to hide the implementation details

Reply:
I've abstracted the is_rest_request to it's own helper which is a definite improvement!

I've kept `is_admin` now as it's used widely so using it in a helper here would make this quite inconsistent. Not opposed to adding it to a helper but I do consider consistency to be more important.